src/main/connector/system/LinuxProcess/LinuxProcess.yaml

```diff
@@ -0,0 +1,102 @@
+extends:
+- ../System/System
+connector:
+  displayName: Linux - Processes (ps)
+  platforms: Any platform with LinuxOS
+  reliesOn: Linux OsCommands
+  information: Gives process specific information and metrics
+  detection:
+    connectionTypes:
+    - remote
+    - local
+    appliesTo:
+    - linux
+    criteria:
+    - type: commandLine
+      commandLine: /usr/bin/which ps || /bin/which ps
+      expectedResult: /usr/bin/ps
```

Comment:
You really have to change this test, as `ps` is sometimes located in `/bin` rather than `/usr/bin`.

Expected result should therefore be `/bin/ps` (which will cover both cases anyway)